lib/services/serviceMapManagement/lib/models/index.d.ts

```diff
@@ -546,15 +622,15 @@ export interface Port extends CoreResource {
  * Represents a collection of clients of a resource. A client group can
  * represent the clients of a port, process, or a machine.
  *
- * @member {object} [clientsOf] Reference to the resource whose clients are
+ * @member {object} clientsOf Reference to the resource whose clients are
  * represented by this group.
  * @member {string} [clientsOf.id] Resource URI.
  * @member {string} [clientsOf.type] Resource type qualifier.
  * @member {string} [clientsOf.name] Resource name.
  * @member {string} [clientsOf.kind] Polymorphic Discriminator
  */
 export interface ClientGroup extends CoreResource {
-  clientsOf?: ResourceReference;
+  clientsOf: ResourceReference;
```

Comment:
Changing a property from optional to required is a breaking change. Requires major version increment.